TaboolaSDKExampleSwift/TaboolaSDKExampleSwift/SDK Native/NativeViewControllerCell.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  NativeViewControllerCell.swift
+//  TaboolaSDKExampleSwift
+//
+//  Created by Karen Shaham Palman on 11/07/2021.
+//  Copyright © 2021 Liad Elidan. All rights reserved.
+//
+
+import UIKit
+import TaboolaSDK
+
+class NativeViewControllerCell: UICollectionViewCell {
+    @IBOutlet weak var tbImageView: TBLImageView!
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var descriptionLabel: TBLDescriptionLabel!
+    @IBOutlet weak var brandingLabel: TBLBrandingLabel!
+    @IBOutlet weak var titleLabel: TBLTitleLabel!
+    @IBOutlet weak var attributionButton: UIButton!
+
+    override func prepareForReuse() {
```

Comment:
Maybe worth explaining.. always think about someone who is less familiar with these APIs